test-specifications/crypto-service/accountUpdateTransaction.md

```diff
@@ -19,111 +17,440 @@ https://github.com/hashgraph/hedera-protobufs/blob/main/services/crypto_update.p
+| status         | string | The status of the submitted `AccountUpdateTransaction` (from a `TransactionReceipt`). |
+
+### Additional Notes
 
-## Properties
+The tests contained in this specification will assume that a valid account was already successfully created. Assume that the account was created with default values, unless specified otherwise. Any `<CREATED_ACCOUNT_ID>` tag will be the account ID of this created account. Any `<PRIVATE_KEY_OF_CREATED_ACCOUNT>` is the DER-encoded hex string of the private key of the account.
 
-### **AccountId:**
+## Property Tests
+
+### **Account ID:**
 
 - The ID of the account to update.
 
-| Test no | Name                                                                             | Input                 | Expected response                                             | Implemented (Y/N) |
-|---------|----------------------------------------------------------------------------------|-----------------------|---------------------------------------------------------------|-------------------|
-| 1       | Update an account with no updates                                                | accountId, privateKey | The account update succeeds                                   | N                 |
-| 2       | Update an account with no updates without signing with the account's private key | accountId             | The account update fails with response code INVALID_SIGNATURE | N                 |
+| Test no | Name                                                                              | Input                                                                                              | Expected response                                                                     | Implemented (Y/N) |
+|---------|-----------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|-------------------|
+| 1       | Updates an account with no updates                                                | accountId=<CREATED_ACCOUNT_ID>, commonTransactionParams.signers=[<PRIVATE_KEY_OF_CREATED_ACCOUNT>] | The account update succeeds.                                                          | N                 |
+| 2       | Updates an account with no updates without signing with the account's private key | accountId=<CREATED_ACCOUNT_ID>                                                                     | The account creation fails with an INVALID_SIGNATURE response code from the network.  | N                 |
+| 3       | Updates an account with no account ID                                             |                                                                                                    | The account creation fails with an INVALID_ACCOUNT_ID response code from the network. | N                 |
```

Comment:
I think should be 'The account update fails' here (also there are similar instances across this file, where 'account creation' should be changed to 'account update').

Reply:
I guess I missed some edits from when I copy-pasted from `accountCreateTransaction.md`.. fixed